### ISSUE TITLE
[Improvement] Improve the clarity of versions of the specification

### ISSUE THREAD

**Issue by eelcodenheijer** [shapeshifter/shapeshifter-specification#88]
## **Describe the improvement**
The documentation of the Shapeshifter specification is not always entirely clear on what version it is supposed to be; for instance: the online browsable specification https://shapeshifter.github.io/shapeshifter-specification/development/ has the `development` version (this is suggested in the URL), but in the upper right corner it denotes `3.0.0` which suggests that this is the current released version. 

There is a pulldown for the version (I think) but the only option is `Development`.

## **Expected behavior**
* Add the version to the generated PDF (preferably in the header or footer); if you are not sure about the version, at least clearly indicate that this is *not* the released version
* Add the released version to the browsable version, clearly indicating that you are looking at the released version (in the upper right corner, and preferably also in the generated PDF)

## **Screenshots**
<img width="1382" alt="Scherm­afbeelding 2023-12-06 om 09 13 00" src="https://github.com/shapeshifter/shapeshifter-specification/assets/4835722/c1a1e57f-d17c-4afe-952f-a51ff50d3c0e">

**Comment by nicorikken** [comment]
Indeed. We need to make a GitHub Release (v.3.0.1, v3.1.0 or v4.0.0) and then the workflow should automatically build and publish a new page.
